### PR TITLE
Fixes libGDX Gradle build error

### PR DESCRIPTION
**Pull request by obigu**
Building libGDX project with Gradle was failing (just run `gradlew clean build` to test) due to a bad workaround I added to fix Android and Desktop tests not running on IDEA (https://github.com/libgdx/libgdx/commit/a30ecf0c1f41f9f2e6a7403520e0bdcd7f4bcb72).

This alternative workaround fixes the Gradle issue and keeps IDEA happy (thanks to PokeMMO).

### CODE REVIEW

**Review by SimonIT — APPROVED** [review]
`gradlew build` works now🎉

File: gdx/build.gradle [hunk 1]
```diff
@@ -42,3 +38,10 @@ test {
         events "passed", "skipped", "failed", "standardOut", "standardError"
     }
 }
+
+// Workaround needed for IDEA to have resources on classpath when running tests (like gdx-tests-lwjgl3)
+task copyIdeaResources(type: Copy) {
+    from "${projectDir}/src/main/resources"
```
Comment (PokeMMO) [inline gdx/build.gradle:44]: Should probably be `${projectDir}/res` for libgdx since it still uses  non-standard folders.

Comment (obigu) [reply gdx/build.gradle]: thanks, your're right, fixed

**Comment by tommyettinger** [comment]
Looks good; I'm glad to hear the Gradle build is working again. I never know which build tool to primarily use with the libGDX repo...

**Pull request merged**